src/components/Notifications/Notifications.js

```diff
@@ -0,0 +1,98 @@
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { childrenPropType } from '../../util/shared-prop-types';
+import { multiplyUnit } from '../../styles/style-helpers';
+import Card from '../Card';
+
+const TOP_LEFT = 'top-left';
+const TOP_RIGHT = 'top-right';
+const BOTTOM_RIGHT = 'bottom-right';
+const BOTTOM_LEFT = 'bottom-left';
+
+const baseStyles = ({ theme, position }) => {
+  const outerSpacing = theme.spacings.giga;
+  const corners = {
+    [TOP_LEFT]: `
+      top: ${outerSpacing};
+      left: ${outerSpacing};
+    `,
+    [TOP_RIGHT]: `
+      top: ${outerSpacing};
+      right: ${outerSpacing};
+    `,
+    [BOTTOM_RIGHT]: `
+      bottom: ${outerSpacing};
+      right: ${outerSpacing};
+    `,
+    [BOTTOM_LEFT]: `
+      bottom: ${outerSpacing};
+      left: ${outerSpacing};
+    `
+  };
+  return css`
+    label: notifications;
+    display: flex;
+    position: fixed;
+    flex-direction: column;
+    width: 400px;
+    max-width: 90vw; ${'' /* FALLBACK: Old Androids don't support calc()  */}
+    max-width: calc(100vw - ${multiplyUnit(outerSpacing, 2)});
```

Comment:
I guess this will go away once we have responsive styles?

Reply:
There is currently no design for these on mobile. This is to prevent the notifications from overflowing outside of the viewport. I don't see why it would need to go away?